lgc/test/CMakeLists.txt

```diff
@@ -42,11 +42,11 @@ configure_lit_site_cfg(
 add_lit_testsuite(check-lgc "Running the LGC regression tests"
   ${CMAKE_CURRENT_BINARY_DIR}
   ${exclude_from_check_all}
-  DEPENDS ${LGC_TEST_DEPENDS}
+  DEPENDS ${LGC_TEST_DEPENDS} llvm-mc
```

Comment:
I think `llvm-mc` can be added to the `LGC_TEST_DEPENDS` list, then it doesn’t need to be repeated

Reply:
Regarding `add_lit_testsuites` dependencies, that's `LLVM_TEST_DEPENDS`, not `LGC_TEST_DEPENDS`. I wonder why would we need dependencies for `add_lit_testsuite` and `add_lit_testsuites` be different. Plus whether we actually need the latter now that we don't really have multiple test suites there.

Reply:
Ah, I missed that. No idea what the `add_lit_testsuites` is for.